code/drasil-gool/lib/GOOL/Drasil/LanguageRenderer/CSharpRenderer.hs

```diff
@@ -121,10 +121,10 @@ instance OOProg CSharpCode where
 
 instance ProgramSym CSharpCode where
   type Program CSharpCode = ProgData
-  prog n st files = do
+  prog n st v files = do
```

Comment:
I'm getting the feeling that there should just be a record passed around for each typeclass as we can probably further add more to the record, but not everything in the record will be used everywhere. For example, license references. @JacquesCarette ?

Reply:
We should definitely be looking at bundling some data together whenever they meaningfully belong as a unit.